homeassistant/components/reolink/binary_sensor.py

```diff
@@ -86,6 +86,7 @@ class ReolinkBinarySensorEntityDescription(
         name="Visitor",
         icon="mdi:bell-ring-outline",
         icon_off="mdi:doorbell",
+        device_class=BinarySensorDeviceClass.OCCUPANCY,
```

Comment:
Is the doorbell really occupied? Sounds like an odd definition to me.
Additionally, we shouldn't use icons when providing a device class.

Reply:
Historically doorbells have used this device class. I agree its a bit odd but its consistant.

I think we should have a DOORBELL device class

Reply:
https://github.com/home-assistant/architecture/discussions/869

Reply:
@bdraco should I just close this PR?
I think the architecture discussion first needs to completed and probably a diffrent implementation than this PR will be chosen...

Reply:
@bdraco @starkillerOG other doorbells are already of that type, it will be new feature and lot of features need to be changed, otherwise this PR can help with their questions and problems with Reolink integration and Reolink Doorbell and integration to home kit because of missing this device class, so please consider this as a two steps change 🙏

Reply:
No, we should not consider this a two-step change @Kepro, as that potentially means we now do something we need to break later on (which is not wished for either). There is no rush in this at all, so let's not rush it.

If you really need it that hard right now, feel free to create your own template sensor that does what you need in the mean time.

../Frenck

Reply:
> you will still need to have a migration phase

I don't follow that.

That said, we won't be adding this one right now. "as it's just one line change" does not justify anything.

../Frenck